packages/turf-buffer/index.js

```diff
@@ -38,8 +38,8 @@ var distanceToRadians = helpers.distanceToRadians;
 module.exports = function (geojson, radius, units, steps) {
     // validation
     if (!geojson) throw new Error('geojson is required');
-    if (!radius) throw new Error('radius is required');
-    if (radius <= 0) throw new Error('radius must be greater than 0');
```

Comment:
Ops! @ralmeidastrider  You are correct, this should allow positive or negative buffer (didn't know `jsts` was able to support this). We should also include this in the test cases.
👍